src/applications/personalization/profile-2/components/connected-apps/components/AppDeletedAlert.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+
+export class AppDeletedAlert extends React.Component {
+  componentDidMount() {
+    this.alert.focus();
+  }
+
+  render() {
+    const { id, accountName, dismissAlert } = this.props;
+    const alertMessage = `${accountName} won't be able to see any new information
```

Comment:
nit: is that a straight single quote? if so it should be a curly (option+shift+])

Reply:
Good eye! Will update!